include/env.bash

```diff
@@ -77,6 +77,8 @@ CB_DB_ENV_DB - Name of the Cloudbreak database
 CB_DB_ENV_PASS - Password for the Cloudbreak database authentication
 CB_DB_ENV_SCHEMA - Used schema in the Cloudbreak database
 CB_DB_ENV_USER - User for the Cloudbreak database authentication
+CB_DB_ENV_SSL - enable of ssl certification or not
```

Comment:
I would describe this variable as: `Enable SSL database connection for Cloudbreak`